dashboard/test/models/pd/application/teacher1819_application_test.rb

```diff
@@ -756,13 +756,31 @@ class Teacher1819ApplicationTest < ActiveSupport::TestCase
     end
 
     test 'locked status appears in csv only when the supplied user can_see_locked_status' do
+      application = build :pd_teacher1819_application
       mock_user = mock
 
       Teacher1819Application.stubs(:can_see_locked_status?).returns(false)
```

Comment:
based on the variables (`header_with_locked`, `row_with_locked`), this stub should return `true`

Reply:
D'oh! Actually the variables are named wrong. The assertions match the stubs, but the variables don't. Will fix.